### PR TITLE
Pull request for Leaderboard setup

### PR DESCRIPTION
**Pull request by TechnoAndy**
## In this project I have implemented the following:

-Used HTML for the setup.
-Created an npm project with webpack.
-Used Gitflow.
-Used supplied wireframe as a guide.
-Used ES6 modules, with import and export.

### CODE REVIEW

File: src/modules/index.html [hunk 1]
```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
```
Comment (ichala) [inline src/modules/index.html]: - [ ] you have some errors where webpack cannot detect your index.js and CSS file , kindly move your index.js, style.css & index.html outside of the module folder, they should be directly listed in the `src` folder, kindly fix it and remove the module folder for now 🙏

File: README.md [hunk 2]
```diff
@@ -1,2 +1,52 @@
...
+
+- HTML
+- CSS
+- Es6
+- Gitflow
+- Visual Studio Code
+
+## Getting Started
+
+- To get a local copy up and running follow these simple example steps.
```
Comment (ichala) [inline README.md:17]: - [ ] to make your repo professional kindly update the Getting Started, Install & Setup sections in your readme file, and try to guide the user on how to install & setup your project like clone, run npm install, etc ...

**Review by ichala — CHANGES_REQUESTED** [review]
Hi @TechnoAndy  , 

Good job so far! 👏👏👏 
There are some issues that you still need to work on to go to the next project but you're almost there!

the-simpsons-homer-simpson

## To Highlight 🎉

- Gitflow followed✔️ 
- No linters Errors ✔️

 ## Required Changes ♻️


- [ ]  **_Check The comments below_**


## Optional suggestions

-N/A

**********
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by brytebee — APPROVED** [review]
Hi, @TechnoAndy👋,

##

### Highlights💡
- Good commit message.✔️ 
- Good PR Summary.✔️
- Great UI design.✔️
- App functions as it should.✔️

##

# STATUS: APPROVED 🕺🥳 

Congratulations🍾🥳!! Your project is ready for presentation!  There is nothing else to say other than... it's time to merge it :shipit: 



Good luck! 🎉 

##

###  **Optional Suggestions:**
- N/A

------

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brytebee in your question so I can receive the notification.
### _**Use @brytebee on slack for faster responses.**_

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**